src/snapshots/new_nu_parser__test__node_output@invalid_types.nu.snap

```diff
@@ -44,22 +45,22 @@ input_file: tests/invalid_types.nu
 4: int
 5: unknown
 6: string
-7: unknown
+7: ()
```

Comment:
I'm wondering if Params should be `forbidden`. They should never be directly evaluated.

Reply:
Yeah that makes sense. As long as we're not doing type *inferencing* we should be good